idaes/models/unit_models/pressure_changer.py

```diff
@@ -342,6 +343,20 @@ def build(self):
         # Construct performance equations
         # Set references to balance terms at unit level
         # Add Work transfer variable 'work'
+        # If the 'work' variable wasn't already built on the control volume but is needed, create it now.
+        if (
+            not hasattr(self.control_volume, "work")
+            and self.config.thermodynamic_assumption == ThermodynamicAssumption.pump
+            and eb is None
+        ):
+            units = self.config.property_package.get_metadata().get_derived_units
+            self.control_volume.work = Var(
```

Comment:
Should we be adding the `work` term to the control volume in this case, or just create `work_mechanical` as a `Var` at the unit level?

Reply:
I started with that approach, but turned out to be a bit cleaner without cascading changes necessary.

Reply:
Left this one open for the time being, subject to other reviewers' thoughts.

Reply:
I'm fine with adding it to the control volume I think.  I suppose it is the expected location for it.

Reply:
@andrewlee94 ,  I looked over this discussion and understood either way above would do the job well.  But while I was reading a initialization API design proposal you wrote,  I would like to say that I think it would be better to only create `work_mechanical`,  as you suggested, by the principle that a var associated with one block should stay with this block, rather than being placed into another block like what is currently.  This way will also be documented more clearly, but I think the doc PR I just opened described the current version well.    Just a thought.  I would be happy to help change it if the IDAES team decides to, otherwise I look forward to concluding the doc PR I opened.  Thanks.